main.py

```
@@ -483,26 +619,76 @@ async def cb_admin_other_functions(callback: CallbackQuery):
+                    admin_menu_msgs[callback.from_user.id] = callback.message.message_id
+                    return
+                if callback.message.caption:
+                    await callback.message.edit_caption(T["admin_panel"], reply_markup=kb_admin())
+                    admin_menu_msgs[callback.from_user.id] = callback.message.message_id
+                    return
+            except TelegramBadRequest:
+                pass
+            try:
+                await callback.message.delete()
+            except Exception:
+                pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)